app/src/main/java/explorer/city/com/cityexplorer/Network/TeleportService.kt

```diff
@@ -0,0 +1,41 @@
+package explorer.city.com.cityexplorer.Network
+
+import com.squareup.moshi.Moshi
+import explorer.city.com.cityexplorer.BaseApplication
+import explorer.city.com.cityexplorer.R
+import io.reactivex.schedulers.Schedulers
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+object TeleportService {
```

Comment:
Maybe you don't, but you currently only use it in one place so you currently don't need a singleton either.